src/context/theme/theme.provider.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react"
+
+import { ThemeContext } from './theme.context'
+
+type ThemeProviderProps = {
+     children: React.ReactNode
+}
+
+export const ThemeProvider: React.FC<ThemeProviderProps> = ({
+     children,
+}: ThemeProviderProps) => {
+     const [theme, setTheme] = useState<boolean>(false)
```

Comment:
The name for this state variable is not appropriate. You're checking if user has selected dark theme or not. So a better name here would be `isDarkTheme`

Reply:
Done